### PR TITLE
Add release notes for ML.NET 0.3

### PR DESCRIPTION
**Pull request by GalOshri**
This adds release notes for ML.NET 0.3

### CODE REVIEW

File: docs/release-notes/0.3/release-0.3.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+```
+
+### Release Notes
+
+Below are some of the highlights from this release.
+
+* Added Field-Aware Factorization Machines (FFM) as a learner for binary classification (#383)
+
+    * FFM is useful for various large sparse datasets, especially in areas such as recommendations and click prediction. It has been used to win various click prediction competitions such as the [Criteo Display Advertising Challenge on Kaggle](https://www.kaggle.com/c/criteo-display-ad-challenge). You can learn more about the winning solution [here](https://www.csie.ntu.edu.tw/~r01922136/kaggle-2014-criteo.pdf).
+    * FFM is a streaming learner so it does not require the entire dataset to fit in memory and can be used as an online learner.
```
Comment (glebuk) [inline docs/release-notes/0.3/release-0.3.md]: >online learner [](start = 114, length = 14)

repetitive.  Make crisper

File: docs/release-notes/0.3/release-0.3.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    * LightLDA is an implementation of [Latent Dirichlet Allocation](https://en.wikipedia.org/wiki/Latent_Dirichlet_allocation) which infers topical structure from text data. 
+    * The implementation of LightLDA in ML.NET is based on [this paper](https://arxiv.org/abs/1412.1576). There is a distributed implementation of LightLDA [here](https://github.com/Microsoft/lightlda).
+
+* Added One-Versus-All (OVA) learner for multiclass classification (#363)
+
+    * [OVA](https://en.wikipedia.org/wiki/Multiclass_classification#One-vs.-rest) (sometimes known as One-Versus-Rest) is an approach to using binary classifiers in multiclass classification problems. 
+    * While some binary classification learners in ML.NET natively support multiclass classification (e.g. Logistic Regression), there are others that do not (e.g. Averaged Perceptron). OVA enables using the latter group for multiclass classification as well.
+
+* Enabled exporting ML.NET models to the [ONNX](https://onnx.ai/) format (#248)
```
Comment (glebuk) [inline docs/release-notes/0.3/release-0.3.md]: > Enabled exporting ML.NET models to the [ONNX](https://onnx.ai/) format [](start = 1, length = 71)

"Enabled export of ML.NET Models to the ONNX..."

File: docs/release-notes/0.3/release-0.3.md (hunk 1 above)
Comment (justinormont) [reply docs/release-notes/0.3/release-0.3.md]: The referenced Kaggle competition seems a bit dated. Do we have any benchmarks showing its current utility vs. other methods?

Comment (GalOshri) [reply docs/release-notes/0.3/release-0.3.md]: Addressed the repetition in next iteration. I am not aware of a more recent published benchmark.

**Pull request merged**